api/v1beta1/frrconfiguration_types.go

```diff
@@ -190,9 +192,16 @@ type Neighbor struct {
 	ToReceive Receive `json:"toReceive,omitempty"`
 
 	// To set if we want to disable MP BGP that will separate IPv4 and IPv6 route exchanges into distinct BGP sessions.
+	// Deprecated: DisableMP is deprecated in favor of DualStackIPFamily.
 	// +optional
 	// +kubebuilder:default:=false
 	DisableMP bool `json:"disableMP,omitempty"`
```

Comment:
should we warn in the webhook when it's specified?

Reply:
won't hurt